extensions/components/com_redcore/site/language/da-DK/da-DK.com_redcore.ini

```diff
@@ -0,0 +1,24 @@
+; redCORE
```

Comment:
you are loading only admin languages here: https://github.com/redCOMPONENT-COM/redCORE/pull/667/files#diff-d231b219ccf80779efeb2e0527693506R34
do you even need those language files?